README.md

```diff
@@ -161,8 +171,11 @@ make u
 ## Environment Variables
 
 | **Environment Variable**|**Description**|**Usage**|**Error Handling**|
```

Comment:
Make sure to leave one character space before and after each header value, otherwise the rendered table looks weird:

```
| **Environment Variable** | **Description** | **Usage** | **Error Handling** |
```